### ISSUE TITLE
Globbing patterns in Exclude

### ISSUE THREAD

**Issue by CupOfTea696** [sstalle/php7cc#134]
Would it be possible to allow globbing patterns in the exclude option?

**Comment by sstalle** [comment]
Yes, that's a good thing to have. Maybe it's finally the time to switch to `symfony/finder` to get sane filtering options...
